api/proquest/importer.py

```diff
@@ -252,10 +263,33 @@ def _get_affiliation_attributes(configuration):
         :return: Configured list of SAML attributes which can contain affiliation ID
         :rtype: List[str]
         """
+        affiliation_attributes = (
+            ProQuestOPDS2ImporterConfiguration.DEFAULT_AFFILIATION_ATTRIBUTES
+        )
+
         if configuration.affiliation_attributes:
-            return configuration.affiliation_attributes
-        else:
-            return ProQuestOPDS2ImporterConfiguration.DEFAULT_AFFILIATION_ATTRIBUTES
+            if isinstance(configuration.affiliation_attributes, list):
+                affiliation_attributes = configuration.affiliation_attributes
+            elif is_string(configuration.affiliation_attributes):
+                affiliation_attributes = tuple(
+                    map(
+                        str.strip,
+                        str(configuration.affiliation_attributes)
+                        .replace("[", "")
+                        .replace("]", "")
+                        .replace("(", "")
+                        .replace(")", "")
+                        .replace("'", "")
+                        .replace('"', "")
+                        .split(","),
```

Comment:
I don't think there's a test for this new code -- the only time I see `affiliation_attributes` being set, it's set to a tuple. Can you add a test?

Reply:
Sure, added additional test cases to `test_checkout_creates_new_token_using_affiliation_id_from_custom_saml_attribute`.